app/src/main/res/layout/activity_accounts.xml

```diff
@@ -0,0 +1,24 @@
+<android.support.v4.widget.DrawerLayout
```

Comment:
This file already present with good naming convention check out https://github.com/openMF/self-service-app/blob/development/app/src/main/res/layout/activity_home.xml.

Remove this file. you are not using anywhere.